### DISCUSSION TITLE
Network Error on creation of account on Self Hosted Server

### DISCUSSION THREAD

**Discussion by vedix95** [stoatchat/discussions#192] [Q&A]
Hello :)
I have to testing Self Hosted Revolt.
But I have an FAILED TO REGISTER! · NETWORK ERROR on registering !!!
I have teste on VPS Server with all port open.
I have test on another computer in localhost (WSL under WIN11) and I have the same problem.

![image](https://user-images.githubusercontent.com/71289366/138592182-b14bd554-9088-46c1-a129-d70e3c853855.png)
Have you any IDEA ??

Thanx :)

**Comment by sussycatgirl** [comment]
I presume that you haven't changed [`.env`](https://github.com/revoltchat/self-hosted/blob/master/.env.example) properly.

The default configuration runs everything on `local.revolt.chat`, if you run it on a VPS you will need to change the addresses in `.env`.

**Reply by vedix95** [reply]
Thanks for your speed reply.
Haaaa Nice Idea !!! I havn't the idea to see on it :)
I'm bad. Thanks ! I test that the next time.
Does The authentification is local or does they Link to an external server ?

**Reply by sussycatgirl** [reply]
> Does The authentification is local or does they Link to an external server ?

Not sure what you mean by that, could you elaborate?

**Comment by demetera** [comment]
I have the same issue, but I'm running instance via nginx on VPS.
in .env file I have
```HOSTNAME=:80```

The rest of URLs in headers refer to full domain name: 
```http://my.domain.com or http://my.domain.com/api (autumn, january, ws)```

In nginx location I have : ```proxy_pass http://localhost:1234;```

Is https://github.com/revoltchat/self-hosted manual incomplete or I forgot something crucial?
Thank
